Connectors/Rhino/Speckle.Connectors.RhinoShared/Bindings/RhinoMapperBinding.cs

```diff
@@ -17,60 +18,112 @@ public record CategoryMapping(
   int ObjectCount
 );
 
+/// <summary>
+/// Represents layers that are all assigned to the same category.
+/// </summary>
+public record LayerCategoryMapping(
+  string CategoryValue,
+  string CategoryLabel,
+  IReadOnlyList<string> LayerIds,
+  IReadOnlyList<string> LayerNames,
+  int LayerCount
+);
+
+/// <summary>
+/// Represents a layer option for the UI dropdown.
+/// </summary>
+public record LayerOption(string Id, string Name);
```

Comment:
extract out

Reply:
[refactor: move records to mapper namespace](https://github.com/specklesystems/speckle-sharp-connectors/pull/1027/commits/9b3bd697eef15a1dbd2408c59b2b55e27d1ce01c)